009_submodules/script.sh

```diff
@@ -12,6 +12,10 @@ git config --global user.name "$GITHUB_USERNAME"
 git config --global advice.detachedHead false
 git config --global push.default current
 
+# why is tree not installed by default?!
+apt-get update -q > /dev/null 2>&1
```

Comment:
I added these lines because `tree` is not available in the image we are using by default. It's not too much of a cost